docs/types.rst

```diff
@@ -259,15 +259,18 @@ Contract Types
 --------------
 
 Every :ref:`contract<contracts>` defines its own type.
-You can implicitly convert contracts to contracts they inherit from,
-and explicitly convert them to and from the ``address`` type, if they have no
-payable fallback functions, or to and from the ``address payable`` type, if they do
-have payable fallback functions.
+You can implicitly convert contracts to contracts they inherit from.
+Contracts can be explicitly converted to and from all other contract types
+and the ``address`` type.
+
+Explicit conversion to and from the ``address payable`` type
+is only possible if the contract type has a payable fallback function.
+The conversion in still performed using ``address(x)`` and not
+using ``address payable(x)``.
```

Comment:
If possible please add a link to the address type in these two paragraphs somewhere.

Reply:
Hm, I really don't see how I should do that, because I think you cannot add a link to a ``-text.

Reply:
I personally have no issue with linking code and have done it before (in Markdown), you could also just add a "Read more in the xx doc" below?

Reply:
I couldn't find out how the syntax for links inside code would be in RST.